website/docs/enterprise/manage-enterprise.md

```diff
@@ -107,9 +107,7 @@ The following events occur when a license expeires and is not renewed within two
 
 License usage is calculated based on total user counts that authentik regularly captures. This data is checked against all valid licenses, and the sum total of all users. Internal and external users are counted based on the number of active users of the respective type saved in authentik. Service account users are not counted towards the license.
 
-:::info
-An **internal** user is typically a team member, such as company employees, who has access to the full Enterprise feature set. An **external** user might be an external consultant, a volunteer in a charitable site, or a B2C customer who logged onto your website to shop. These users don't get access to Enterprise features.
-:::
+An **internal** user is typically a team member, such as company employees, who has access to the full Enterprise feature set. An **external** user might be an external consultant, a volunteer in a charitable site, or a B2C customer who logged onto your website to shop. External users don't get access to Enterprise features, nor to the **My applications** page in authentik. Instead, external users are authetnticated and then redirected to log directly into their [default application](../core/brands.md#external-user-settings).
```

Comment:
Nit: Object pluralization doesn't agree between root and adverbial clauses..  Suggestion: "... typically a team member, such as a company employee...."  The examples for external users are all singular, and correct.